forem/forem
# [15 min fix] Add aria-current attribute to current dashboard link

## What type of PR is this? (check all applicable)


## Description

This introduces the `aria-current="page"` attribute to the dashboard link that's currently active (on the dashboards page in desktop views).

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

- On a desktop, visit `/dashboard`
- Click through the links on the left-hand navigation
- For each, check that when the link is visible, and it's the currently displayed page, the attribute `aria-current="page"` is present in the HTML (or use a screen reader and check that the link is announced as the current page)
- Check that the non-active links do not have the `aria-current` attribute / are not announced via screen reader as the current page


![Screenshot showing the following tags link selected, and the dev tool inspector with the aria-current attribute highlighted](https://user-images.githubusercontent.com/20773163/124298765-8d57a800-db54-11eb-8277-0c2fff52f0ad.png)


### UI accessibility concerns?

This fix makes navigation between dashboards more user-friendly for screen reader users

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: Just a small 15 min fix
